elixir/apps/domain/lib/domain/accounts.ex

```diff
@@ -18,6 +19,17 @@ defmodule Domain.Accounts do
     end
   end
 
+  def all_active_paid_accounts! do
+    ["Team", "Enterprise"]
+    |> Enum.flat_map(&all_active_accounts_by_type!/1)
+  end
+
+  def all_active_accounts_by_type!(account_type) do
```

Comment:
We might add a `type` field on account and then it will be a confusing name:
```suggestion
  def all_active_accounts_by_subscription_name!(subscription_name) do
```

Reply:
Sounds good.  This has been updated.

Out of curiosity, is there a reason we didn't put a `type` field on the account already?

Reply:
@bmanifold did not have a need so far. But as complexity grows (eg. we have multi-account users) we might change that